CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## Release 1.0.0
```

Comment:
Probably want this to be `0.1.0` for the initial release https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

Reply:
I have the initial release at `1.0.0` since I'm assuming once this is in it's immediately going to be shipped with Bolt and be ready for use in production: https://semver.org/#how-do-i-know-when-to-release-100 . If we don't immediately ship this and are still considering it 'in development', then I'm fine going down to a `0.x` release.

Reply:
Right but _we_ aren't currently using it in production, and it seems better to make as few guarantees as possible.

Reply:
I don't really understand what you mean by we aren't using it in production...
Regardless, updated to 0.1.0.